### ISSUE TITLE
add-on doesn't appear to do anything (Firefox 60)

### ISSUE THREAD

**Issue by boldcompany** [mikeconley/multi-select-tabs#33]
The MultiSelect Tabs add-on doesn't appear to do anything in Firefox 60.

**Comment by deltab** [comment]
Does it show up in the sidebar menu, between 'Synced Tabs' and 'Move Sidebar to Left'? It does for me in Firefox 61.

**Comment by boldcompany** [comment]
Yes, it does. So MultiSelect Tabs operations cannot occur in the main Firefox tab bar?
